google_compute_engine_oslogin/Makefile

```diff
@@ -7,7 +7,8 @@ MINOR = 2
 REVISION = 0
 
 LIBNSS_OSLOGIN_CACHE = libnss_$(BASENAME)_cache
-LIBNSS_OSLOGIN_CACHE_NAME = $(LIBNSS_OSLOGIN_CACHE).so.2
+LIBNSS_OSLOGIN_CACHE_NAME = libnss_$(NAME)-cache-$(MAJOR).$(MINOR).$(REVISION).so
```

Comment:
NAME is defined as: NAME = google-compute-engine-$(BASENAME)
So if I use '_' then it would become something like libnss_google-compute-engine-oslogin_cache-1.3.0.so
Not sure if this looks better...

Reply:
After changing the name format to libnss_cache_*, this is fixed.